lib/transformations/resolve/resolve.js

```diff
@@ -11,12 +14,12 @@ module.exports = function transformer(j, ast) {
 			}, -1);
 	};
 
-	const isModulePresent = p => {
+	const isModulePresent = (p: IPath<IProperty<IObjectExpression>>) => {
 		const modules = p.node.value.properties.filter(prop => prop.key.name === 'modules');
 		return modules.length > 0 && modules[0];
 	};
 
-	const createModuleArray = p => {
+	const createModuleArray = (p: IPath<IProperty<IObjectExpression>>): IPath<IProperty<IObjectExpression>> => {
```

Comment:
Any way to shorten down line length?

Reply:
We can just say that `p` is a generic `IPath<*>` but this won't give us the specific type. There's no harm but if we stick with types, considering the depth no interfaces that we could have due to the AST, cases like this might happen sometime

Reply:
I was more thinking of moving some of the code, if doable, to a new line 👍